test/test-pipe-bind-error.c

```diff
@@ -137,3 +137,19 @@ TEST_IMPL(pipe_listen_without_bind) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+TEST_IMPL(pipe_bind_or_listen_error_after_close) {
+  uv_pipe_t server;
+
+  ASSERT(uv_pipe_init(uv_default_loop(), &server, 0) == 0);
+  uv_close((uv_handle_t *)&server, NULL);
```

Comment:
Style nit (for all new tests)
```suggestion
  uv_close((uv_handle_t*) &server, NULL);
```

Secondly, could you write the new tests with the new `ASSERT_EQ()` macro family?

Reply:
Thanks! done.